client/signup/steps/design-type-with-store/index.jsx

```diff
@@ -21,7 +21,6 @@ import PressableStoreStep from './pressable-store';
 import { BlogImage, PageImage, GridImage, StoreImage } from '../design-type-with-store/type-images';
 import { abtest } from 'lib/abtest';
 import { setDesignType } from 'state/signup/steps/design-type/actions';
-
```

Comment:
There's a ton of these empty-line between imports fixes in files unrelated to codemod's other modifications.

Should we let them be or remove empty lines only from files we modify with codemod?

Reply:
Hmm, I'd keep the scope of the PR as focused/limited as possible, so I'd rather not include otherwise unchanged files like these...